guide/14-deep-learning/object_detection_and_tracking_on_videos.ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>Since this topic is broken into its own page here, add context by saying what the object detection page does (works on images) and contrast this page by saying works on video files and performs two types of operations - detection and tracking. This part can go before the Object Detection heading.</p><p>----------</p><p>Since you have note imported <code>predict_video</code> until now, when talking about it for the first time, use its fully qualified namespace as <code>arcgis.learn.SingleShotDetector.predict_video()</code> and hyper link to https://developers.arcgis.com/python/api-reference/arcgis.learn.html#arcgis.learn.SingleShotDetector.predict_video</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>predict video function is also supported for RetinaNet, so i 'll add hyperlink to api reference</p>